Form-Controls/index.html

```diff
@@ -14,14 +14,51 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+    <div> 
+      <label for="name">Name</label>
+      <input type="text" id="name" placeholder="Enter Your Name" pattern="^[a-zA-Z]+(?:[-' ][a-zA-Z]+)*$" required>
+    </div>
+    <div> 
+      <label for="email">Email</label>
+      <input type="email" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$" id="email" placeholder="Enter Your e-mail" required>
+    </div>
+    <div>
+    Choose your T-shirt's colour
+        <div>
+            <label for="Red">Red</label>
+            <input type="radio" name="colour" id="Red" value="Red" required>
+        </div>
+        <div>
+            <label for="Green">Green</label>
+            <input type="radio" name="colour" id="Green" value="Green">
+        </div>
+        <div>
+            <label for="White">White</label>
+            <input type="radio" name="colour" id="White" value="White">
+        </div>
+        Choose your T-shirt's Size
+        <div>
+          <label for="Size">T-shirt's Size</label>
```

Comment:
Same here, you could use a better semantic approach to better structure your HTML markup.